README.md

```diff
@@ -26,6 +26,16 @@ Try `cn.dl.elvish.io` if you are in China and the download is too slow. (In the
 
 See also [Building Elvish](#building-elvish).
 
+### RPM package
+
+RPM Package for Fedora is available in [FZUG Repo](https://github.com/FZUG/repo/wiki/Add-FZUG-Repository). You can install it by add FZUG repo first:
```

Comment:
**the** FZUG repo; by **adding**.

I think it is better to just say: "First add the FZUG repo", since there is no doubt that you are giving instructions on how to install elvish; the syntax is also consistent with "Then install elvish",